### PR TITLE
Bug - 5652 - tc report favicons

### LINKED ISSUE #5652: 🐛 Bug - Favicons are 404ing on tc-report
## 🐛 Bug

Favicons are still 404'ing for tc-report. Currently, they're not under the /tc-report path. Add the  favicons under the /tc-report path (or use the ones from the main app).

## 🙋‍♀️ Proposed Solution

Add favicons under tc-report path OR use ones in main app.

## ✅ Acceptance Criteria

- [ ] Favicons show up on tc-report pages

### PR DESCRIPTION
**Pull request by yonikid15**
🤖 Resolves #5652.

## 👋 Introduction

Fixes favicons 404'ing on `tc-report` app. Moves the favicons from the `/tc-report` to `/tc-report/assets` path. The changes I made are all under [e9634a3](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5840/commits/e9634a3dc149024ef4ff843d7b468a47862041b1).

## 🧪 Testing

1. Go to `tc-report` and ensure favicons are rendering correctly.

## 📸 Screenshot

image

### CODE REVIEW

File: maintenance/scripts/refresh_api.sh [hunk 1]
```diff
@@ -1,9 +1,9 @@
 #! /bin/bash
 
-parent_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+parent_path=~/gc-digital-talent/maintenance/scripts
```
Comment (mnigh) [inline maintenance/scripts/refresh_api.sh:3]: Was this change on purpose or just testing? It breaks the script for me locally and seems out of scope for this issue.

File: maintenance/scripts/refresh_api.sh [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 source ${parent_path}/lib/common.sh
 
-cd /var/www/html/api
+cd ~/gc-digital-talent/api
```
Comment (mnigh) [inline maintenance/scripts/refresh_api.sh:6]: Was this change on purpose or just testing? It breaks the script for me locally and seems out of scope for this issue.

File: tc-report/_includes/head.html [hunk 3]
```diff
@@ -26,11 +26,11 @@
...
   <!-- Favicons -->
-  <link rel="apple-touch-icon" sizes="180x180" href="{{ '/apple-touch-icon.png' | prepend: site.baseurl }}">
-  <link rel="icon" type="image/png" sizes="32x32" href="{{ '/favicon-32x32.png' | prepend: site.baseurl }}">
-  <link rel="icon" type="image/png" sizes="16x16" href="{{ '/favicon-16x16.png' | prepend: site.baseurl }}">
-  <link rel="manifest" href="{{ '/site.webmanifest' | prepend: site.baseurl }}">
-  <link rel="mask-icon" href="{{ '/safari-pinned-tab.svg' | prepend: site.baseurl }}" color="#5bbad5">
+  <link rel="apple-touch-icon" sizes="180x180" href="{{ '/apple-touch-icon.png' | prepend: site.asset_url }}">
+  <link rel="icon" type="image/png" sizes="32x32" href="{{ '/favicon-32x32.png' | prepend: site.asset_url }}">
+  <link rel="icon" type="image/png" sizes="16x16" href="{{ '/favicon-16x16.png' | prepend: site.asset_url }}">
+  <link rel="manifest" href="{{ '/site.webmanifest' | prepend: site.asset_url }}">
```
Comment (mnigh) [inline tc-report/_includes/head.html:32]: This change makes sense and points to the correct webmanifest (previously was pointing to [this one](https://github.com/GCTC-NTGC/gc-digital-talent/blob/main/apps/web/public/site.webmanifest)). However, the correct site.webmanifest is missing some information and has invalid file paths. You can test this in [Firefox](https://firefox-source-docs.mozilla.org/devtools-user/application/manifests/index.html). This should do the trick for English (we have an [issue open](https://github.com/GCTC-NTGC/gc-digital-talent/issues/4780) for French and other locales):

```
{
    "name": "GC Digital Talent",
    "short_name": "GC Digital Talent",
    "icons": [
        {
            "src": "/tc-report/assets/android-chrome-192x192.png",
            "sizes": "192x192",
            "type": "image/png"
        },
        {
            "src": "/tc-report/assets/android-chrome-512x512.png",
            "sizes": "512x512",
            "type": "image/png"
        }
    ],
    "theme_color": "#ffffff",
    "background_color": "#ffffff",
    "display": "standalone"
}
```

**Review by mnigh — CHANGES_REQUESTED** [review]
🆒 work, I like the favicon and stuff being in the assets directory. I've left a few questions and comments.

File: maintenance/scripts/refresh_api.sh (hunk 1 above)
Comment (yonikid15) [reply maintenance/scripts/refresh_api.sh]: Hmm, I thought I changed this back with this commit 😕  https://github.com/GCTC-NTGC/gc-digital-talent/pull/5840/commits/0c2dc4d57518c4b3366e7f52fa8118143a2c0ac5. Here is the fix again [19071bf](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5840/commits/19071bf51947d7318eaf34077b8499a0d84260de)

File: tc-report/_includes/head.html (hunk 3 above)
Comment (yonikid15) [reply tc-report/_includes/head.html:32]: Awesome thanks Matt, I'll add the changes in.

Comment (yonikid15) [reply tc-report/_includes/head.html:32]: Changes added [71db81d](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5840/commits/71db81d1d7cd01fb4f64db1f73c037820b1579be)

**Review by mnigh — APPROVED** [review]
nice work! 👍
